### ISSUE TITLE
Android/iOS: Accessibility - expand label/name for password generator toggles

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/mobile#1918]
### Steps To Reproduce

1. Turn on VoiceOver/iOS, Talkback/Android
2. Go to Password generator screen
3. Go to the "A-Z", "a-z", "0-9", "!@#$%^&*" toggles

### Expected Result

The switches should have a clear announcement that is understandable for assistive technology users.

### Actual Result

In Talkback/Android, the options are announced as "a to zed", "a to zed" (no differentiation on upper or lowercase), "zero to nine", "at hash dollar percent caret and asterisk".

In VoiceOver/iOS, the options are announced as "a zed", "a zed", "zero nine", "exclamation point at hash symbol dollar percent caret and star".

### Additional Context

Suggest doing the same change made on the extension https://github.com/bitwarden/clients/pull/2233, the desktop app https://github.com/bitwarden/desktop/pull/1461, and the web vault https://github.com/bitwarden/web/pull/1611 - expanding the accessible names/labels for those switches to `Uppercase (A-Z)`, `Lowercase (a-z)`, `Numbers (0-9)`, `Special Characters (!@#$%^&*)`

Obviously, this will mean adding strings in the app resources, and getting them translated as well (translations have already been done for the browser, desktop, and web vault, so should be straightforward to just grab them from there).

### Operating System

Android, iOS

### Build Version

2.18.0 (4572)

### Beta

- [ ] Using a pre-release version of the application.

**Comment by patrickhlauke** [comment]
I'm guessing the changes need to be made in https://github.com/bitwarden/mobile/blob/master/src/iOS.Core/Controllers/PasswordGeneratorViewController.cs and https://github.com/bitwarden/mobile/blob/master/src/App/Pages/Generator/GeneratorPage.xaml

**Comment by patrickhlauke** [comment]
thanks @andrebispo5
